stats/views.py

```diff
@@ -311,6 +365,38 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class FragmentTableViewMDS(FragmentTableView):
+    def get_queryset(self):
```

Comment:
If at all possible, it would maybe be nicer if you would set the fragment_pks already in the MDS view, right? When you execute the `reduce_model` function maybe?

Reply:
This is a bit confusing, because that feature is built in a weird way IMO. It relies on a form being POSTed with the fragment id(s) that correspond to the point the user clicked on (https://github.com/UUDigitalHumanitieslab/timealign/blob/feature/new-mds-view/stats/views.py#L248 https://github.com/UUDigitalHumanitieslab/timealign/blob/feature/new-mds-view/stats/templates/stats/mds.html#L84)
I don't think it really belongs on the session but I didn't want to change it too much. Anyway I can't set it in `reduce_model` because it relies on input from the user.